package.json

```diff
@@ -66,7 +66,8 @@
     "koa2-cors": "^2.0.6",
     "lodash": "^4.17.15",
     "log4js": "^6.7.1",
-    "mongoose": "6.*",
+    "mongodb": "~5.9.1",
```

Comment:
❓ why so strict? why not "^5.9.1"

Reply:
I see in the [compatibility table of mongo](https://www.mongodb.com/docs/drivers/node/current/compatibility/#compatibility-table-legend) that they change support (upwards) using minor versions for the driver, so I opted to keep the minor version constant for orka as well. I changed it to `^` to be less restrictive.